include/openssl/ssl.h

```diff
@@ -328,8 +328,9 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 # define SSL_OP_TLSEXT_PADDING                           0x00000010U
 /* Reserved value (until OpenSSL 3.0.0)                  0x00000020U */
 # define SSL_OP_SAFARI_ECDHE_ECDSA_BUG                   0x00000040U
+/* Reserved value (until OpenSSL 3.0.0)                  0x00000080U */
+# define SSL_OP_ALLOW_SSL_EOF                            0x00000080U
```

Comment:
All these "Reserved value (until OpenSSL 3.0.0)" comments can be removed. The next release *is* 3.0 and therefore these values are no longer reserved!! They were only there for ABI compatibility - but we're not concerned with ABI compatibility for 3.0.

Reply:
Done.